pom.xml

```diff
@@ -39,7 +39,7 @@
   </developers>
 
   <properties>
-    <changelist>999999-SNAPSHOT</changelist>
+    <changelist>68126-SNAPSHOT-NOWATCHER</changelist>
```

Comment:
Revert this. If tests pass in the PR, and it is up to date with `master`, you will get an incremental deployment automatically (link displayed among status checks).

Reply:
(Currently there are four failures in https://ci.jenkins.io/job/Plugins/job/kubernetes-plugin/job/PR-1167/6/testReport/org.csanchez.jenkins.plugins.kubernetes.pipeline/KubernetesPipelineTest/.)

Reply:
sorry I was on PTO, done